openapi/endpoints/funding-channel/funding-channel-create.yaml

```diff
@@ -0,0 +1,50 @@
+post:
+  tags:
+    - Funding Channel
+  summary: Claim a funding channel
+  description: |
+    This endpoint can be used to claim an on chain funding channel.
+  requestBody:
+    content:
+      application/json:
+        schema:
+          $ref: "./models/create-funding-channel.yaml"
+    required: true
+
+  responses:
+    "200":
+      description: Successful operation
+      content:
+        application/json:
+          schema:
+            $ref: "./models/get-funding-channel.yaml"
+    "403":
+      description: |
+        User is not allowed to perform the action with the reason stated in the `errorCode`
+
+        **FORBIDDEN**
+        Insufficient permissions to perform the action.
+        
+        **ACCOUNT_TYPE_INVALID**
+        `accountType` of the `accountId` is not of type `PARTNER`.
+
+        **FUNDING_CHANNEL_EXISTS**
+        Instance of this FundingChannel has already been claimed.
+        
+        **STORAGE_ADDRESS_INVALID**
```

Comment:
this can occur if the client sends a non valid ethereum address eg `fakeaddress123` or a valid ethereum address but its not an instance created from the fundsManager. Is this description enough for the two instances or do we want to be more specific? @javcode